R/utilities.R

```diff
@@ -48,6 +48,10 @@ ifNotNull <- function(condition, outputIfNotNull, outputIfNull = NULL) {
 #' See https://github.com/Open-Systems-Pharmacology/OSPSuite-R/issues/476 for details
 #' @import stringr
 encodeUnit <- function(unit) {
+  
+  #TODO maybe we don't even need to load from .NET

```

Comment:
Oh you have it here... See my suggestion above.

Reply:
I don't understand what you mean